mzLib/FlashLFQ/FlashLfqEngine.cs

```diff
@@ -685,6 +685,7 @@ private void QuantifyMatchBetweenRunsPeaks(SpectraFileInfo idAcceptorFile)
 
                             // default range (if only 1 datapoint, or SD is 0, range is very high, etc)
                             double rtRange = MbrRtWindow;
+                            bool interquartileRangeUsed = false;
 
                             if (nearbyCalibrationPoints.Count < 6 && nearbyCalibrationPoints.Count > 1 && rtDiffs.StandardDeviation() > 0)
                             {
```

Comment:
Why stdev * 6 and IQR * 4.5 here? I'm not sure I understand those constants.

Reply:
I honestly don't know. That's how FlashLFQ is written, and those values aren't explained. Rob might remember